src/bun.js/node/dir_iterator.zig

```diff
@@ -184,8 +184,16 @@ pub fn NewIterator(comptime use_windows_ospath: bool) type {
             }
         },
         .windows => struct {
+            const FILE_DIRECTORY_INFORMATION = std.os.windows.FILE_DIRECTORY_INFORMATION;
+            const FILE_DIRECTORY_INFORMATION_PTR = *align(2) FILE_DIRECTORY_INFORMATION;
```

Comment:
I think a more descriptive comment here would be nice, or a link to the Zig PR

I like the one from the Zig PR:
```
// While the official api docs guarantee FILE_BOTH_DIR_INFORMATION to be aligned properly
// this may not always be the case (e.g. due to faulty VM/Sandboxing tools)
```

Reply:
added